packages/suite/src/views/wallet/tokens/TokensNavigation.tsx

```diff
@@ -65,17 +70,25 @@ export const TokensNavigation = ({
         <Row alignItems="center" justifyContent="space-between">
             <SubTabs activeItemId={routeName} size="medium">
                 <SubTabs.Item
-                    id="wallet-tokens"
+                    id={isNft ? 'wallet-nfts' : 'wallet-tokens'}
                     iconName="tokens"
-                    onClick={goToRoute('wallet-tokens')}
-                    count={tokens.shownWithBalance.length}
+                    onClick={goToRoute(isNft ? 'wallet-nfts' : 'wallet-tokens')}
+                    count={
+                        isNft
+                            ? tokens.shownWithBalance.length + tokens.shownWithoutBalance.length
```

Comment:
why both?

Reply:
how does it behave when you send nft out of account? is it in data with quantity 0?

Reply:
this was caused by a mistake in getTokens, fixed now